pages/moderation/editsubmission/index.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
do you still need these 3 pages?

Reply:
I believe so. I tried removing different ones. Seems seems like every folder that is a page needs its on index to create the `/param` and if I want to add an id as the param it needs the file with the name of the `[param]`

Reply:
hmmmmmmmmmmmmmmmmmm.,..... I'm wondering if we actually need it. I'm guessing you normally would for building out static pages through nuxt+vue router. However, since these pages aren't indexed or publicly visible, I'm wondering if there's a programmatic way to a subscribe them in vue router without requiring pages.

Reply:
https://nuxt.com/docs/getting-started/routing

Seems like you are right and I need to refactor and see how to get it working 🙇

Reply:
Done!